src/tpt_reports/tpt_reports.py

```diff
@@ -60,7 +144,11 @@ def main() -> None:
                 error="Possible values for --log-level are "
                 + "debug, info, warning, error, and critical.",
             ),
+            "SERVICENOW_ID": Use(str, error="SERVICENOW_ID must be a string."),
+            "ELECTION_NAME": Use(str, error="ELECTION_NAME must be a string."),
+            "DOMAIN_TESTED": Use(str, error="DOMAIN_TESTED must be a string."),
```

Comment:
Do you do any validation anywhere that the string in `"SERVICENOW_ID"` looks like a valid ID? 
 Same question for the `"DOMAIN_TESTED"` string- is it syntactically a valid domain?

Reply:
I like this idea, but see both as a future PRs. 
- As `SERVICENOW_ID` needs better definition clarified, noted in this Issue: Remove 3rd party reference from arguments #30, we can look to come to a method to validate.
- For `DOMAIN_TESTED`, created an Issue to support the syntact issue and/but also possibly expand on the scope where we could use a service like "WhoIs" to validate its existence, and possibly gather relevant data to improve upon the report. Validate DOMAIN_TESTED argument inputs #36

Reply:
@schmelz21 Makes sense.  Can you please add `TODO:` comment(s) in here that point to those issues?

Reply:
TODO comments have been added for `SERVICENOW_ID` and `DOMAIN_TESTED` in commits: 1a54afee764ce69e38c92fd97c6c058d7840d877 and fb290fdc0698f002cf17f2cfaaa152d7fe9ff50b .